### PR TITLE
Add article about volume groups

### PR DESCRIPTION
**Pull request by houtknots**

### CODE REVIEW

File: _posts/volume/2023-03-12-volume-groups.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+Sadly at the time of writing this article 03-2024 the OpenStack does not
+provide a way to restore a volume group snapshot. The snapshots are only 
+restorable individually. We recommend using the OpenStack Dashboard to restore
+a volume group snapshot or using the OpenStack CLI to restore the snapshots
+individually.
+
+You can find the OpenStack Snapshot in the OpenStack Dashboard by going to the
+`VOLUMES` section and then clicking on `Snapshots`.
+
+<!-- TODO: Add a link to a snapshot restore article >
```
Comment (RobertJansen1) [inline _posts/volume/2023-03-12-volume-groups.md:332]: Please remove this line in the final article

Comment (RobertJansen1) [inline _posts/volume/2023-03-12-volume-groups.md:332]: Please remove the todo from the final article

**Review by RobertJansen1 — CHANGES_REQUESTED** [review]

File: _posts/volume/2023-03-12-volume-groups.md (hunk 1 above)
Comment (houtknots) [reply _posts/volume/2023-03-12-volume-groups.md:332]: I fixed the comment so it does not show on the page.
I do think we should keep it there so we can find it in the future.

Comment (RobertJansen1) [reply _posts/volume/2023-03-12-volume-groups.md:332]: works for me :)

**Comment by houtknots** [comment]
> Create a volume group using the OpenStack CLI
> 
> openstack volume group create --os-volume-api-version 3.51 --volume-group-type --availability-zone --volume-type
> 
> --name has to be added in front of to give the group a name

Fixed

**Review by RobertJansen1 — APPROVED** [review]

**Pull request merged**